docs/containers.conf.5.md

```diff
@@ -9,9 +9,10 @@ Container engines like Podman & Buildah read containers.conf file, if it exists
 and modify the defaults for running containers on the host. containers.conf uses
 a TOML format that can be easily modified and versioned.
 
-Container engines read the /usr/share/containers/containers.conf and
-/etc/containers/containers.conf, and /etc/containers/containers.conf.d/*.conf files
-if they exist. When running in rootless mode, they also read
+Container engines read the /usr/share/containers/containers.conf,
+/etc/containers/containers.conf, and /etc/containers/containers.conf.d/*.conf
+files if they exist.
+When running in rootless mode, they also read
 $HOME/.config/containers/containers.conf and
 $HOME/.config/containers/containers.conf.d/*.conf files.
```

Comment:
Not in your edits but I see a Markdown formatting issue with

/etc/containers/containers.conf.d/*.conf

I think there is a backslash missing. So instead it should be

/etc/containers/containers.conf.d/\*.conf

There is also another one

$HOME/.config/containers/containers.conf.d/*.conf files.

Reply:
Fixed.